ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -49,6 +51,16 @@ var supportedComposeYamlOptions = []string{
 
 var supportedComposeYamlOptionsMap = getSupportedComposeYamlOptionsMap()
 
+type ECSFields struct {
+	Version        string
+	TaskDefinition ecsTaskDef `yaml:"task_definition"`
+}
+
+type ecsTaskDef struct {
+	NetworkMode string `yaml:"ecs_network_mode"`
+	TaskRoleArn string `yaml:"task_role_arn"`
+}
+
```

Comment:
Question: This is only a subset of the proposed changes, but we could push these changes by themselves right?

Reply:
Yes, if you recall our discussion in person last week, I wanted to push these changes first just to get it out and I'll submit the other part (adding `essential` separately).